test/Concurrency/actor_isolation_swift6.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-swift-frontend -disable-availability-checking -swift-version 6 -emit-sil -o /dev/null -verify %s
+// RUN: %target-swift-frontend -disable-availability-checking -swift-version 6 -emit-sil -o /dev/null -verify -enable-experimental-feature GlobalActorIsolatedTypesUsability %s
 
 // REQUIRES: concurrency
```

Comment:
Note that you'll need to add `asserts` to the `REQUIRES:` list because the experimental feature is only enable-able in asserts compilers.